matlab_simulation/05-estimation/kalman_filter/measurement_model.m

```diff
@@ -0,0 +1,53 @@
+function [C,D,Q,m] = measurement_model(example)
```

Comment:
This function works fine, but I question its necessity.

To me, these functions (motion_model too) simply serve as an initialization for the simulation, where there is no commonality between the examples. The code is not too complex, which is why I would recommend use moving the code into each of their respective examples.

Furthermore, If someone wanted to change something small in the in the examples they would need to hunt down this function and change it in here because the inputs to the function do not allow you to adjust parameters like the state space matrices.